app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1345,22 +1314,29 @@ DeviceDelegateOpenXR::CreateLayerEquirect(const VRLayerPtr &aSource) {
 
 VRLayerPassthroughPtr
 DeviceDelegateOpenXR::CreateLayerPassthrough() {
-  if (!m.layersEnabled || !m.passthroughSupported || !m.passthrough) {
+  if (!m.layersEnabled || !m.passthroughStrategy->usesCompositorLayer()) {
```

Comment:
nit: would make sense to add a check to assert the passthoughStrategy instance is valid ? It seems you do it on other methods as well (eg,  usesPassthroughCompositorLayer)